test/cypress/integration/agroup-contributions.spec.js

```diff
@@ -0,0 +1,565 @@
+const userId = Math.floor(Math.random() * 10000)
+const groupName = 'Dreamers'
+const usersDisplayName = {
+  1: 'Greg',
+  2: 'Margarida',
+  3: 'Pierre',
+  4: 'Sandrina'
+}
+
```

Comment:
This file seems to have been duplicated and added on accident? Can it be removed?

Reply:
Yea sure - renaming a specific test file this way is a method I use sometimes to have it run first